app/frontend/src/internationalization/trans/chefs/en/en.json

```diff
@@ -862,7 +865,7 @@
       "fetchFormFieldsErrMsg": "An error occurred while fetching the list of fields for this form.",
       "fetchFormFieldsConsErrMsg": "Error getting form {formId}: {error}",
       "publishDraftErrMsg": "An error occurred while publishing.",
-      "publishDraftConsErrMsg": "Error publishing ${draftId}: ${error}",
+      "publishDraftConsErrMsg": "Error publishing {draftId}: {error}",
```

Comment:
Removed these extra `$` characters - these were sometimes also being copied over to the translations, but I think those have already been fixed.